cernopendata/modules/fixtures/data/docs/atlas-higgs-machine-learning-challenge-2015/atlas-higgs-machine-learning-challenge-2015.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "title": "ATLAS Higgs Machine Learning Challenge",
+    "slug": "higgs-machine-learning-challenge-2015",
```

Comment:
Please keep the slugs under which the news articles are visible on PROD now, so that we don't have to make any redirections so soon after the release.

E.g. `atlas-higgs-machine-learning-challenge` for this one... You can check the other ones at http://opendata.web.cern.ch/search?page=1&size=20&type=News.

We won't have dirname-vs-slug harmony, similarly to "About CMS", but since we haven't got to change news to docs before the release, let's stick to the released slugs...

Reply:
fixed now 👍  dirnames are consistent imo, but slugs are contextual 😄